pkg/components/state/loader.go

```diff
@@ -47,4 +48,7 @@ func Load() {
 	RegisterStateStore("gcp.firestore", func() state.Store {
 		return firestore.NewFirestoreStateStore()
 	})
+	RegisterStateStore("azure.sqlserver", func() state.Store {
+		return sqlserver.NewSQLServerStateStore()
```

Comment:
As this is not Azure specific, can we change to just `"sqlserver"`?

Reply:
Yes, it makes sense. What about the components-contrib project? Should I:
- keep as /state/azure/
- move it to /state/
- move it to /state/microsoft/
- move it to /state/sql/sqlserver

Reply:
/state/sqlserver